lib/SIL/SILFunction.cpp

```diff
@@ -405,17 +405,28 @@ TargetFunction("view-cfg-only-for-function", llvm::cl::init(""),
                llvm::cl::desc("Only print out the cfg for this function"));
 #endif
 
-
-void SILFunction::viewCFG() const {
+namespace {
+void viewCFGHelper(const SILFunction* F, bool skipBBContents) {
```

Comment:
Why not just make this a static function? Then you can eliminate the namespace, i.e.:

```
static void viewCFGHelper(const SILFunction *f, bool skipBBComments) { ... }
```

Also, please downcase the f.

Reply:
Done.